From c607266619c5ab78ad5d4179b3ea93cfb6348391 Mon Sep 17 00:00:00 2001
From: Andrea Bolognani <abologna@redhat.com>
Date: Mon, 3 May 2021 09:06:34 +0200
Subject: [PATCH] meson: Fix compatibility with Meson 0.58

Builds failed with

  tests/meson.build:690:0: ERROR: List item must be one
  of <class 'str'>, not <class 'list'>

before this change.

https://gitlab.com/libvirt/libvirt/-/issues/158

Upstream-Status: Backport [https://gitlab.com/libvirt/libvirt/-/commit/c607266619c5ab78ad5d4179b3ea93cfb6348391]

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Andrea Bolognani <abologna@redhat.com>
Reviewed-by: Peter Krempa <pkrempa@redhat.com>
Reviewed-by: Pavel Hrdina <phrdina@redhat.com>
---
 tests/meson.build | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/meson.build b/tests/meson.build
index 05c3e90195..9900983d0c 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -687,12 +687,12 @@ foreach name : test_scripts
   test(name, script, env: tests_env)
 endforeach
 
+testenv = runutf8
+testenv += 'VIR_TEST_FILE_ACCESS=1'
+
 add_test_setup(
   'access',
-  env: [
-    'VIR_TEST_FILE_ACCESS=1',
-    runutf8,
-  ],
+  env: testenv,
   exe_wrapper: [ python3_prog, check_file_access_prog.path() ],
 )
 
-- 
GitLab

